hub/core/chunk_engine.py

```diff
@@ -451,6 +451,10 @@ def _create_new_chunk(self):
         chunk.key = chunk_key
         return chunk
 
+    def append(self, sample):
+        """Formats a single `sample` (compresseses/decompresses if applicable) and feeds it into `_append_bytes`."""
+        self.extend([sample])
```

Comment:
why do we need append method in chunk engine? Tensor will already handle this and always send a list to extend